docker/kubernetes-tentacle/bootstrapRunner/bootstrapRunner.go

```diff
@@ -79,6 +79,9 @@ func main() {
 		fmt.Fprintln(os.Stderr, "bootstrapRunner.go: Failed to execute bootstrap script", err)
 	}
 
+	stdoutLogFile.Flush()
```

Comment:
I think it might be worth adding a comment saying these are here "just in case"

Reply:
done